io/TextReader.cpp

```diff
@@ -131,7 +131,12 @@ void TextReader::initialize(PointTableRef table)
     if (!m_istream)
         throwError("Unable to open text file '" + m_filename + "'.");
 
+
     std::string header;
+
+    // Skip any lines requested
+    for (uint32_t i = 0; i < m_headerSkip; ++i)
```

Comment:
This logic needs to be repeated in ready(), or some other accommodation needs to be made.